CRITICAL - 2024-04-22 23:33:18 --> CodeIgniter\Database\Exceptions\DataException: There is no data to update.
in SYSTEMPATH\BaseModel.php on line 970.
 1 SYSTEMPATH\BaseModel.php(970): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
 2 SYSTEMPATH\Model.php(807): CodeIgniter\BaseModel->update([...], [])
 3 APPPATH\Controllers\Home.php(100): CodeIgniter\Model->update('28', [...])
 4 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Home->acceptRequest()
 5 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(85): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\laragon\\www\\CAPSTONEBACKEND\\public\\index.php')
CRITICAL - 2024-04-22 23:33:23 --> CodeIgniter\Database\Exceptions\DataException: There is no data to update.
in SYSTEMPATH\BaseModel.php on line 970.
 1 SYSTEMPATH\BaseModel.php(970): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
 2 SYSTEMPATH\Model.php(807): CodeIgniter\BaseModel->update([...], [])
 3 APPPATH\Controllers\Home.php(100): CodeIgniter\Model->update('28', [...])
 4 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Home->acceptRequest()
 5 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(85): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\laragon\\www\\CAPSTONEBACKEND\\public\\index.php')
CRITICAL - 2024-04-22 23:33:24 --> CodeIgniter\Database\Exceptions\DataException: There is no data to update.
in SYSTEMPATH\BaseModel.php on line 970.
 1 SYSTEMPATH\BaseModel.php(970): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
 2 SYSTEMPATH\Model.php(807): CodeIgniter\BaseModel->update([...], [])
 3 APPPATH\Controllers\Home.php(100): CodeIgniter\Model->update('28', [...])
 4 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Home->acceptRequest()
 5 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(85): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\laragon\\www\\CAPSTONEBACKEND\\public\\index.php')
CRITICAL - 2024-04-22 23:35:45 --> ParseError: syntax error, unexpected single-quoted string "created_at", expecting "]"
in APPPATH\Models\PWDModel.php on line 15.
 1 SYSTEMPATH\Autoloader\Autoloader.php(288): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\laragon\\www\\CAPSTONEBACKEND\\app\\Models\\PWDModel.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(266): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Models\\PWDModel')
 3 APPPATH\Controllers\Home.php(91): CodeIgniter\Autoloader\Autoloader->loadClass('App\\Models\\PWDModel')
 4 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Home->acceptRequest()
 5 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(85): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\laragon\\www\\CAPSTONEBACKEND\\public\\index.php')
CRITICAL - 2024-04-22 23:39:02 --> ErrorException: Undefined property: stdClass::$id
in APPPATH\Controllers\Home.php on line 87.
 1 APPPATH\Controllers\Home.php(87): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: stdClass::$id', 'C:\\laragon\\www\\CAPSTONEBACKEND\\app\\Controllers\\Home.php', 87)
 2 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Home->accept()
 3 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(85): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\laragon\\www\\CAPSTONEBACKEND\\public\\index.php')
CRITICAL - 2024-04-22 23:39:41 --> ErrorException: Undefined property: stdClass::$id
in APPPATH\Controllers\Home.php on line 87.
 1 APPPATH\Controllers\Home.php(87): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: stdClass::$id', 'C:\\laragon\\www\\CAPSTONEBACKEND\\app\\Controllers\\Home.php', 87)
 2 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Home->accept()
 3 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(85): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\laragon\\www\\CAPSTONEBACKEND\\public\\index.php')
CRITICAL - 2024-04-22 23:39:42 --> ErrorException: Undefined property: stdClass::$id
in APPPATH\Controllers\Home.php on line 87.
 1 APPPATH\Controllers\Home.php(87): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: stdClass::$id', 'C:\\laragon\\www\\CAPSTONEBACKEND\\app\\Controllers\\Home.php', 87)
 2 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Home->accept()
 3 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(85): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\laragon\\www\\CAPSTONEBACKEND\\public\\index.php')
CRITICAL - 2024-04-22 23:39:42 --> ErrorException: Undefined property: stdClass::$id
in APPPATH\Controllers\Home.php on line 87.
 1 APPPATH\Controllers\Home.php(87): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: stdClass::$id', 'C:\\laragon\\www\\CAPSTONEBACKEND\\app\\Controllers\\Home.php', 87)
 2 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Home->accept()
 3 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(85): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\laragon\\www\\CAPSTONEBACKEND\\public\\index.php')
CRITICAL - 2024-04-22 23:39:42 --> ErrorException: Undefined property: stdClass::$id
in APPPATH\Controllers\Home.php on line 87.
 1 APPPATH\Controllers\Home.php(87): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: stdClass::$id', 'C:\\laragon\\www\\CAPSTONEBACKEND\\app\\Controllers\\Home.php', 87)
 2 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Home->accept()
 3 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(85): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\laragon\\www\\CAPSTONEBACKEND\\public\\index.php')
CRITICAL - 2024-04-22 23:44:24 --> ParseError: syntax error, unexpected single-quoted string "created_at", expecting "]"
in APPPATH\Models\PWDModel.php on line 15.
 1 SYSTEMPATH\Autoloader\Autoloader.php(288): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\laragon\\www\\CAPSTONEBACKEND\\app\\Models\\PWDModel.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(266): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Models\\PWDModel')
 3 APPPATH\Controllers\Home.php(92): CodeIgniter\Autoloader\Autoloader->loadClass('App\\Models\\PWDModel')
 4 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Home->acceptRequest()
 5 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(85): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\laragon\\www\\CAPSTONEBACKEND\\public\\index.php')
CRITICAL - 2024-04-22 23:46:17 --> ErrorException: Undefined property: stdClass::$money_requestsId
in APPPATH\Controllers\Home.php on line 95.
 1 APPPATH\Controllers\Home.php(95): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: stdClass::$money_requestsId', 'C:\\laragon\\www\\CAPSTONEBACKEND\\app\\Controllers\\Home.php', 95)
 2 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Home->acceptRequest()
 3 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(85): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\laragon\\www\\CAPSTONEBACKEND\\public\\index.php')
ERROR - 2024-04-22 23:47:11 --> mysqli_sql_exception: Unknown column 'status' in 'field list' in C:\laragon\www\CAPSTONEBACKEND\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\laragon\www\CAPSTONEBACKEND\system\Database\MySQLi\Connection.php(306): mysqli->query('UPDATE `money_r...', 0)
#1 C:\laragon\www\CAPSTONEBACKEND\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `money_r...')
#2 C:\laragon\www\CAPSTONEBACKEND\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `money_r...')
#3 C:\laragon\www\CAPSTONEBACKEND\system\Database\BaseBuilder.php(2469): CodeIgniter\Database\BaseConnection->query('UPDATE `money_r...', Array, false)
#4 C:\laragon\www\CAPSTONEBACKEND\system\Model.php(393): CodeIgniter\Database\BaseBuilder->update()
#5 C:\laragon\www\CAPSTONEBACKEND\system\BaseModel.php(987): CodeIgniter\Model->doUpdate(Array, Array)
#6 C:\laragon\www\CAPSTONEBACKEND\system\Model.php(807): CodeIgniter\BaseModel->update(Array, Array)
#7 C:\laragon\www\CAPSTONEBACKEND\app\Controllers\Home.php(129): CodeIgniter\Model->update('28', Array)
#8 C:\laragon\www\CAPSTONEBACKEND\system\CodeIgniter.php(941): App\Controllers\Home->rejectRequest()
#9 C:\laragon\www\CAPSTONEBACKEND\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#10 C:\laragon\www\CAPSTONEBACKEND\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\laragon\www\CAPSTONEBACKEND\public\index.php(85): CodeIgniter\CodeIgniter->run()
#12 C:\laragon\www\CAPSTONEBACKEND\system\Commands\Server\rewrite.php(47): require_once('C:\\laragon\\www\\...')
#13 {main}
CRITICAL - 2024-04-22 23:47:11 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'status' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(2469): CodeIgniter\Database\BaseConnection->query('UPDATE `money_requests` SET `status` = :status:
WHERE `money_requests`.`id` IN :money_requests.id:', [...], false)
 2 SYSTEMPATH\Model.php(393): CodeIgniter\Database\BaseBuilder->update()
 3 SYSTEMPATH\BaseModel.php(987): CodeIgniter\Model->doUpdate([...], [...])
 4 SYSTEMPATH\Model.php(807): CodeIgniter\BaseModel->update([...], [...])
 5 APPPATH\Controllers\Home.php(129): CodeIgniter\Model->update('28', [...])
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Home->rejectRequest()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(85): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\laragon\\www\\CAPSTONEBACKEND\\public\\index.php')
CRITICAL - 2024-04-22 23:47:11 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'status' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `money_requests` SET `status` = \'rejected\'
WHERE `money_requests`.`id` IN (\'28\')')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `money_requests` SET `status` = \'rejected\'
WHERE `money_requests`.`id` IN (\'28\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2469): CodeIgniter\Database\BaseConnection->query('UPDATE `money_requests` SET `status` = :status:
WHERE `money_requests`.`id` IN :money_requests.id:', [...], false)
 4 SYSTEMPATH\Model.php(393): CodeIgniter\Database\BaseBuilder->update()
 5 SYSTEMPATH\BaseModel.php(987): CodeIgniter\Model->doUpdate([...], [...])
 6 SYSTEMPATH\Model.php(807): CodeIgniter\BaseModel->update([...], [...])
 7 APPPATH\Controllers\Home.php(129): CodeIgniter\Model->update('28', [...])
 8 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Home->rejectRequest()
 9 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
10 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(85): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\laragon\\www\\CAPSTONEBACKEND\\public\\index.php')
CRITICAL - 2024-04-22 23:47:11 --> [Caused by] mysqli_sql_exception: Unknown column 'status' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('UPDATE `money_requests` SET `status` = \'rejected\'
WHERE `money_requests`.`id` IN (\'28\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `money_requests` SET `status` = \'rejected\'
WHERE `money_requests`.`id` IN (\'28\')')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `money_requests` SET `status` = \'rejected\'
WHERE `money_requests`.`id` IN (\'28\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2469): CodeIgniter\Database\BaseConnection->query('UPDATE `money_requests` SET `status` = :status:
WHERE `money_requests`.`id` IN :money_requests.id:', [...], false)
 5 SYSTEMPATH\Model.php(393): CodeIgniter\Database\BaseBuilder->update()
 6 SYSTEMPATH\BaseModel.php(987): CodeIgniter\Model->doUpdate([...], [...])
 7 SYSTEMPATH\Model.php(807): CodeIgniter\BaseModel->update([...], [...])
 8 APPPATH\Controllers\Home.php(129): CodeIgniter\Model->update('28', [...])
 9 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Home->rejectRequest()
10 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
11 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(85): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\laragon\\www\\CAPSTONEBACKEND\\public\\index.php')
ERROR - 2024-04-22 23:47:12 --> mysqli_sql_exception: Unknown column 'status' in 'field list' in C:\laragon\www\CAPSTONEBACKEND\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\laragon\www\CAPSTONEBACKEND\system\Database\MySQLi\Connection.php(306): mysqli->query('UPDATE `money_r...', 0)
#1 C:\laragon\www\CAPSTONEBACKEND\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `money_r...')
#2 C:\laragon\www\CAPSTONEBACKEND\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `money_r...')
#3 C:\laragon\www\CAPSTONEBACKEND\system\Database\BaseBuilder.php(2469): CodeIgniter\Database\BaseConnection->query('UPDATE `money_r...', Array, false)
#4 C:\laragon\www\CAPSTONEBACKEND\system\Model.php(393): CodeIgniter\Database\BaseBuilder->update()
#5 C:\laragon\www\CAPSTONEBACKEND\system\BaseModel.php(987): CodeIgniter\Model->doUpdate(Array, Array)
#6 C:\laragon\www\CAPSTONEBACKEND\system\Model.php(807): CodeIgniter\BaseModel->update(Array, Array)
#7 C:\laragon\www\CAPSTONEBACKEND\app\Controllers\Home.php(129): CodeIgniter\Model->update('28', Array)
#8 C:\laragon\www\CAPSTONEBACKEND\system\CodeIgniter.php(941): App\Controllers\Home->rejectRequest()
#9 C:\laragon\www\CAPSTONEBACKEND\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#10 C:\laragon\www\CAPSTONEBACKEND\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\laragon\www\CAPSTONEBACKEND\public\index.php(85): CodeIgniter\CodeIgniter->run()
#12 C:\laragon\www\CAPSTONEBACKEND\system\Commands\Server\rewrite.php(47): require_once('C:\\laragon\\www\\...')
#13 {main}
CRITICAL - 2024-04-22 23:47:12 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'status' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(2469): CodeIgniter\Database\BaseConnection->query('UPDATE `money_requests` SET `status` = :status:
WHERE `money_requests`.`id` IN :money_requests.id:', [...], false)
 2 SYSTEMPATH\Model.php(393): CodeIgniter\Database\BaseBuilder->update()
 3 SYSTEMPATH\BaseModel.php(987): CodeIgniter\Model->doUpdate([...], [...])
 4 SYSTEMPATH\Model.php(807): CodeIgniter\BaseModel->update([...], [...])
 5 APPPATH\Controllers\Home.php(129): CodeIgniter\Model->update('28', [...])
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Home->rejectRequest()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(85): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\laragon\\www\\CAPSTONEBACKEND\\public\\index.php')
CRITICAL - 2024-04-22 23:47:12 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'status' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `money_requests` SET `status` = \'rejected\'
WHERE `money_requests`.`id` IN (\'28\')')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `money_requests` SET `status` = \'rejected\'
WHERE `money_requests`.`id` IN (\'28\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2469): CodeIgniter\Database\BaseConnection->query('UPDATE `money_requests` SET `status` = :status:
WHERE `money_requests`.`id` IN :money_requests.id:', [...], false)
 4 SYSTEMPATH\Model.php(393): CodeIgniter\Database\BaseBuilder->update()
 5 SYSTEMPATH\BaseModel.php(987): CodeIgniter\Model->doUpdate([...], [...])
 6 SYSTEMPATH\Model.php(807): CodeIgniter\BaseModel->update([...], [...])
 7 APPPATH\Controllers\Home.php(129): CodeIgniter\Model->update('28', [...])
 8 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Home->rejectRequest()
 9 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
10 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(85): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\laragon\\www\\CAPSTONEBACKEND\\public\\index.php')
CRITICAL - 2024-04-22 23:47:12 --> [Caused by] mysqli_sql_exception: Unknown column 'status' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('UPDATE `money_requests` SET `status` = \'rejected\'
WHERE `money_requests`.`id` IN (\'28\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `money_requests` SET `status` = \'rejected\'
WHERE `money_requests`.`id` IN (\'28\')')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `money_requests` SET `status` = \'rejected\'
WHERE `money_requests`.`id` IN (\'28\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2469): CodeIgniter\Database\BaseConnection->query('UPDATE `money_requests` SET `status` = :status:
WHERE `money_requests`.`id` IN :money_requests.id:', [...], false)
 5 SYSTEMPATH\Model.php(393): CodeIgniter\Database\BaseBuilder->update()
 6 SYSTEMPATH\BaseModel.php(987): CodeIgniter\Model->doUpdate([...], [...])
 7 SYSTEMPATH\Model.php(807): CodeIgniter\BaseModel->update([...], [...])
 8 APPPATH\Controllers\Home.php(129): CodeIgniter\Model->update('28', [...])
 9 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Home->rejectRequest()
10 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
11 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(85): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\laragon\\www\\CAPSTONEBACKEND\\public\\index.php')
ERROR - 2024-04-22 23:47:12 --> mysqli_sql_exception: Unknown column 'status' in 'field list' in C:\laragon\www\CAPSTONEBACKEND\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\laragon\www\CAPSTONEBACKEND\system\Database\MySQLi\Connection.php(306): mysqli->query('UPDATE `money_r...', 0)
#1 C:\laragon\www\CAPSTONEBACKEND\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `money_r...')
#2 C:\laragon\www\CAPSTONEBACKEND\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `money_r...')
#3 C:\laragon\www\CAPSTONEBACKEND\system\Database\BaseBuilder.php(2469): CodeIgniter\Database\BaseConnection->query('UPDATE `money_r...', Array, false)
#4 C:\laragon\www\CAPSTONEBACKEND\system\Model.php(393): CodeIgniter\Database\BaseBuilder->update()
#5 C:\laragon\www\CAPSTONEBACKEND\system\BaseModel.php(987): CodeIgniter\Model->doUpdate(Array, Array)
#6 C:\laragon\www\CAPSTONEBACKEND\system\Model.php(807): CodeIgniter\BaseModel->update(Array, Array)
#7 C:\laragon\www\CAPSTONEBACKEND\app\Controllers\Home.php(129): CodeIgniter\Model->update('28', Array)
#8 C:\laragon\www\CAPSTONEBACKEND\system\CodeIgniter.php(941): App\Controllers\Home->rejectRequest()
#9 C:\laragon\www\CAPSTONEBACKEND\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#10 C:\laragon\www\CAPSTONEBACKEND\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\laragon\www\CAPSTONEBACKEND\public\index.php(85): CodeIgniter\CodeIgniter->run()
#12 C:\laragon\www\CAPSTONEBACKEND\system\Commands\Server\rewrite.php(47): require_once('C:\\laragon\\www\\...')
#13 {main}
CRITICAL - 2024-04-22 23:47:12 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'status' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(2469): CodeIgniter\Database\BaseConnection->query('UPDATE `money_requests` SET `status` = :status:
WHERE `money_requests`.`id` IN :money_requests.id:', [...], false)
 2 SYSTEMPATH\Model.php(393): CodeIgniter\Database\BaseBuilder->update()
 3 SYSTEMPATH\BaseModel.php(987): CodeIgniter\Model->doUpdate([...], [...])
 4 SYSTEMPATH\Model.php(807): CodeIgniter\BaseModel->update([...], [...])
 5 APPPATH\Controllers\Home.php(129): CodeIgniter\Model->update('28', [...])
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Home->rejectRequest()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(85): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\laragon\\www\\CAPSTONEBACKEND\\public\\index.php')
CRITICAL - 2024-04-22 23:47:12 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'status' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `money_requests` SET `status` = \'rejected\'
WHERE `money_requests`.`id` IN (\'28\')')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `money_requests` SET `status` = \'rejected\'
WHERE `money_requests`.`id` IN (\'28\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2469): CodeIgniter\Database\BaseConnection->query('UPDATE `money_requests` SET `status` = :status:
WHERE `money_requests`.`id` IN :money_requests.id:', [...], false)
 4 SYSTEMPATH\Model.php(393): CodeIgniter\Database\BaseBuilder->update()
 5 SYSTEMPATH\BaseModel.php(987): CodeIgniter\Model->doUpdate([...], [...])
 6 SYSTEMPATH\Model.php(807): CodeIgniter\BaseModel->update([...], [...])
 7 APPPATH\Controllers\Home.php(129): CodeIgniter\Model->update('28', [...])
 8 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Home->rejectRequest()
 9 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
10 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(85): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\laragon\\www\\CAPSTONEBACKEND\\public\\index.php')
CRITICAL - 2024-04-22 23:47:12 --> [Caused by] mysqli_sql_exception: Unknown column 'status' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('UPDATE `money_requests` SET `status` = \'rejected\'
WHERE `money_requests`.`id` IN (\'28\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `money_requests` SET `status` = \'rejected\'
WHERE `money_requests`.`id` IN (\'28\')')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `money_requests` SET `status` = \'rejected\'
WHERE `money_requests`.`id` IN (\'28\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2469): CodeIgniter\Database\BaseConnection->query('UPDATE `money_requests` SET `status` = :status:
WHERE `money_requests`.`id` IN :money_requests.id:', [...], false)
 5 SYSTEMPATH\Model.php(393): CodeIgniter\Database\BaseBuilder->update()
 6 SYSTEMPATH\BaseModel.php(987): CodeIgniter\Model->doUpdate([...], [...])
 7 SYSTEMPATH\Model.php(807): CodeIgniter\BaseModel->update([...], [...])
 8 APPPATH\Controllers\Home.php(129): CodeIgniter\Model->update('28', [...])
 9 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Home->rejectRequest()
10 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
11 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(85): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\laragon\\www\\CAPSTONEBACKEND\\public\\index.php')
